docs/assemblies/con_provisioning-bare-metal-data-plane-nodes.adoc

```diff
@@ -53,8 +53,9 @@ Sufficient number of edpm node BareMetalHost(BMH) CRs should be created and be i
 `Available` state (after inspection).By default baremetal-operator would be looking
 for BMHs in the `openshift-machine-api` namespace.
 
-If BMHs in other namesapces are expected to be managed, `Provisioning` resource should
-be patched to watch all namespaces with `watchAllNamesapces: true`.
+`Provisioning` resource should be patched to watch all namespaces with `watchAllNamesapces: true`
```

Comment:
```suggestion
`Provisioning` resource should be patched to watch all namespaces with `watchAllNamespaces: true`
```

Reply:
Thanks, I'm blind and assumed what's there aready is correct :smile: